apps/api/src/secret/secret.e2e.spec.ts

```diff
@@ -840,10 +840,8 @@ describe('Secret Controller Tests', () => {
 
     expect(response.statusCode).toBe(200)
     expect(response.json().length).toBe(1)
-
     const secret = response.json()[0]
```

Comment:
- Check if the 0th element contains `environmentName` and `environmentId`
- Rename `secret` in `const secret = response.json()[0]` to something more meaningful.